src/environments/environment.ts

```diff
@@ -15,7 +15,7 @@ export const environment: IEnv = {
   baseHref: '',
   isUserNameAutocompleteEnabled: true,
   showTokenExpirationCustomization: true,
-  mockApi: false
+  mockApi: true
```

Comment:
Is this intended?

Reply:
this one does not really make a difference for prod build as it's only used for local development. No issue checking it in